### DISCUSSION TITLE
Somthing wrong

### DISCUSSION THREAD

**Discussion by venividivici666** [AmirMardan/PyFWI#10] [Q&A]
Hello! When I pip install PyFWI and test the fwi_example.ipynb under PyTorch, an error will occurr says "ModuleNotFoundError: No module named 'pyopencl'", how can I deal with it? Thank you very much!
![image](https://github.com/AmirMardan/PyFWI/assets/60508154/e6fb834f-4f4c-42f5-8765-3145578891bb)

**Comment by AmirMardan** [comment]
Hi,

You just need to install PyOpenCL before running the example. Please use
`pip install pyopencl`.
